File: app/src/main/java/zone/com/zanimate/entity/Property.java
package zone.com.zanimate.entity;

/**
 * Created by fuzhipeng on 16/10/4.
 */

public class Property {
    private int x;
    private int y;
    private int z;
    private int xR;
    private int yR;
    private int zR;
    private boolean alignCenter;

    public int getX() {
        return x;
    }

    public void setX(int x) {
        this.x = x;
    }

    public int getY() {
        return y;
    }

    public void setY(int y) {
        this.y = y;
    }

    public int getZ() {
        return z;
    }

    public void setZ(int z) {
        this.z = z;
    }

    public int getxR() {
        return xR;
    }

    public void setxR(int xR) {
        this.xR = xR;
    }

    public int getyR() {
        return yR;
    }

    public void setyR(int yR) {
        this.yR = yR;
    }

    public int getzR() {
        return zR;
    }

    public void setzR(int zR) {
        this.zR = zR;
    }

    public boolean isAlignCenter() {
        return alignCenter;
    }

    public void setAlignCenter(boolean alignCenter) {
        this.alignCenter = alignCenter;
    }
}

File: app/src/main/java/zone/com/zanimate/entity/Property.kt
package zone.com.zanimate.entity

/**
 * Created by fuzhipeng on 16/10/4.
 */

class Property {
    var x: Int = 0
    var y: Int = 0
    var z: Int = 0
    private var xR: Int = 0
    private var yR: Int = 0
    private var zR: Int = 0
    var isAlignCenter: Boolean = false

    fun getxR(): Int {
        return xR
    }

    fun setxR(xR: Int) {
        this.xR = xR
    }

    fun getyR(): Int {
        return yR
    }

    fun setyR(yR: Int) {
        this.yR = yR
    }

    fun getzR(): Int {
        return zR
    }

    fun setzR(zR: Int) {
        this.zR = zR
    }
}
